openstack/cc3test/alerts/ccloud-hermes.alerts

```diff
@@ -3,7 +3,7 @@ groups:
   rules:
   - alert: CCloudHermesApiDown
     expr: |
-        cc3test_status{service="hermes", type="api"} == 0
+        max(cc3test_status{service="hermes", type="api"} == 0) unless max(cc3test_status{service="keystone", type="api"} == 0)
```

Comment:
using the `max` statement here will filter out all the labels attached to `cc3test_status`. Meaning, everything with `{{ $labels.... }} will not work anymore.

I don't see any other mappings here but
```
type: api
service: hermes
```

which can be replaced in the alert definition.